docker-compose.yml

```
@@ -27,8 +27,10 @@ services:
       - "7687:7687"
     volumes:
       - neo4j_data:/data
+    env_file:
+      - .env.db
     environment:
-      NEO4J_AUTH: neo4j/changeme # Default user/password, change in production!
+      NEO4J_AUTH: ${NEO4J_USER}/${NEO4J_PASSWORD} # Default user/password, change in production!
```

YAMLlint (1.37.1)

[warning] too few spaces before comment: expected 2

(comments)